File: Famoser.ExpenseMonitor.Webpage/models/entities/ExpenseEntity.php
<?php
/**
 * Created by PhpStorm.
 * User: Florian Moser
 * Date: 11.01.2016
 * Time: 12:42
 */

namespace famoser\expensemonitor\webpage\models\entities;


use famoser\expensemonitor\webpage\models\Expense;

class ExpenseEntity extends Expense
{
    public function __construct(Expense $note)
    {
        //$this->Id = $ds->Id; Id censored
        $this->Guid = $note->Guid;
        $this->Description = $note->Description;
        $this->CreateTime = $note->CreateTime;
        $this->Amount = $note->Amount;
    }
}

File: Famoser.ExpenseMonitor.Data/Entities/ExpenseEntity.cs
using System;
using System.Runtime.Serialization;

namespace Famoser.ExpenseMonitor.Data.Entities
{
    [DataContract]
    public class ExpenseEntity
    {
        [DataMember]
        public Guid Guid;

        [DataMember]
        public string Description;

        [DataMember]
        public DateTime CreateTime;

        [DataMember]
        public double Amount;
    }
}
